src/sidecar/sidecarManager.ts

```diff
@@ -379,17 +379,54 @@ export class SidecarManager {
         // Set up the environment for the sidecar process.
         const sidecar_env = constructSidecarEnv(process.env);
 
+        const stderrPath = `${SIDECAR_LOGFILE_PATH}.stderr`;
         try {
+          // try to create a file to track any stderr output from the sidecar process
+          fs.writeFileSync(stderrPath, "");
+          const stderrFd = fs.openSync(stderrPath, "w");
+
           const sidecarProcess = spawn(executablePath, [], {
             detached: true,
-            stdio: "ignore",
+            // ignore stdin/stdout, pipe stderr to the file
+            stdio: ["ignore", "ignore", stderrFd],
             env: sidecar_env,
           });
           logger.info(
             `${logPrefix}: started sidecar process with pid ${sidecarProcess.pid}, logging to ${sidecar_env["QUARKUS_LOG_FILE_PATH"]}`,
           );
           sidecarProcess.unref();
```

Comment:
Maybe ... we should grab the `pid` from sidecarProcess into a new const _before_ the unref() call? I would not be colored surprised if `unref()` did a lot to actively forget, but I haven't read its docs today.